accord-core/src/main/java/accord/local/cfk/CommandsForKey.java

```diff
@@ -740,7 +753,7 @@ public enum InternalStatus
         STABLE(true, false),
         APPLIED(true, false),
         INVALIDATED(false, false),
-        TRUNCATED_OR_PRUNED(false, false)
```

Comment:
Truncation is normal truncation, and I have stopped encoding truncation in the CFK. Basically the CFK continues retaining any truncated transaction as its pre-truncation state. 

Pruned on the other hand means we have decided the CFK probably doesn't need to know about the transaction anymore, so we remove it. This may happen to a truncated or non-truncated transaction. If we later witness it as a dependency, we may have to load it from disk to see what we knew about it. So we trade smaller memory footprint for occasionally having to consult command state.